BricksAndTiles.podspec

```
@@ -0,0 +1,24 @@
+Pod::Spec.new do |s|
+    s.name                  = 'BricksAndTiles'
+    s.version               = '0.2.0'
```

The correct current version would be `0.0.3`, right?
```suggestion
    s.version               = '0.0.3'
```